src/libraries/System.Private.Xml/src/System/Xml/Serialization/IXmlSerializable.cs

```diff
@@ -12,7 +13,7 @@ namespace System.Xml.Serialization
     /// </devdoc>
     public interface IXmlSerializable
     {
-        XmlSchema GetSchema();
+        XmlSchema? GetSchema();
         void ReadXml(XmlReader reader);
         void WriteXml(XmlWriter writer);
```

Comment:
I agree but I'd still prefer for this to be more constrained for 3 reasons:
- it feels like it has more sense for this to be non-null (what would this method do when null provided?)
- for parity with ReadXml
- we can relax this in the future (also since interface is public people might implement it without handling null and having it being nullable won't harm)

Reply:
> (what would this method do when null provided?)
Seems like it just does some logging at the start and at the end.

OK, that's fair, thanks for clarifying.